### PR TITLE
Add 'CLSAG' Moneropedia entry

### LINKED ISSUE #1141: Add 'CLSAG' Moneropedia entry

### PR DESCRIPTION
**Pull request by erciccione**
Closes #1141

### CODE REVIEW

File: _i18n/en/resources/moneropedia/clsag.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+terms: ["CLSAG"]
+summary: Monero's efficient way of building ring signatures
+---
+
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+CLSAG (Concise Linkable Ring Signatures) is the new way Monero builds @ring-signatures, which replaces MLSAG (multilayered Linkable Spontaneous Anonymous Group Signature). CLSAG went live and mandatory with the [Monero network upgrade of October 2020]({{ site.baseurl }}/2020/08/18/network-upgrade-october-2020.html). CLSAG maintains the same functionalities of MLSAG, but with the advantage of considerable size savings for @transactions and improvements in verification speed, which result in smaller and faster transaction creation and verification. This new technology was developed by contributors of the [Monero Research Lab]({{ site.baseurl }}/resources/research-lab/) and [audited by external researchers]({{ site.baseurl_root }}/resources/research-lab/audits/clsag.pdf).
```
Comment (SarangNoether) [inline _i18n/en/resources/moneropedia/clsag.md:7]: CLSAG (Concise Linkable Spontaneous Anonymous Group) signatures are...

**Review by SarangNoether — CHANGES_REQUESTED** [review]

**Comment by erciccione** [comment]
Rebased and adapted to https://github.com/monero-project/monero-site/pull/1198 (`clsag` only to the English yaml file)

**Review by SamsungGalaxyPlayer — CHANGES_REQUESTED** [review]
Looks awesome, though can we also include the CLSAG logo at the bottom from https://github.com/monero-project/monero-site/blob/master/blog/assets/clsag/clsag.svg

**Comment by erciccione** [comment]
Would be nice to have some colour in the page, but adding an image would be inconsistent with the other entries, don't know if the change worths it.

**Comment by SamsungGalaxyPlayer** [comment]
Is there a significant reason not to? If a diagram or something would be useful in another Moneropedia entry, I would support its inclusion.

**Comment by erciccione** [comment]
@SamsungGalaxyPlayer I did a couple of tests. If we want the image to be centered, we need to add an html block. I would avoid that because it would complicate things when will be time to feed these documents to weblate. Would be different for a diagram, since it wouldn't need to be centered IMO. I would prefer to avoid this logo since it's more a complication than anything. We also link from this page to the blog post announcing CLSAG, which have the image right at the top.

**Review by SamsungGalaxyPlayer — APPROVED** [review]

**Pull request merged**